### DISCUSSION TITLE
Can OpenPNM and Porespy use GPU?

### DISCUSSION THREAD

**Discussion by javierpagalo** [PMEAL/porespy#645] [Q&A] [answered]
Is there any way to use GPU in OpenPNM or Porespy?

**Comment by jgostick** [comment, marked as answer]
Using the GPU is not as simple as it seems.  Each function needs to be coded to specifically leverage it, and only certain functions can actually benefit from it.  So the answer is no, not really.  If you urgently need it, you can always inspect our source code and translate it to GPU.  Libraries like numba and pycuda offer support for many numpy functions, and the list is always growing.
